src/components/navigation-bar/navigation-bar.jsx

```diff
@@ -1,63 +1,94 @@
-import PropTypes from "prop-types";
-import { Navbar, Nav, Button } from "react-bootstrap";
+import { Navbar, Nav, Button, Container, Form } from "react-bootstrap";
 import { Link } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { logoutUser } from "../../redux/reducers/userData";
+import PropTypes from "prop-types";
+import { OutsideAlerter } from "./navbarWrapper";
+
+export const NavigationBar = ({ isLoading }) => {
+  const token = useSelector((state) => state.userData.token);
+  const dispatch = useDispatch();
+
+  const handleToggle = () => {
+    if (
+      !document.getElementById("navbar-toggler").classList.contains("collapsed")
+    ) {
+      document.getElementById("navbar-toggler").click();
+    }
+  };
```

Comment:
Using React means that you can do this with the React state instead of using vanilla Javascript.

Reply:
Thanks for pointing that out, using React states improved that part of the code a lot!